### PR TITLE
PBjs Core Testing: update Windows 10 testing on Chrome to v 89.0

### PR DESCRIPTION
**Pull request by ChrisHuie**
Update windows 10 testing on chrome from version 80.0 to 89.0. There also still remains testing in this environment for chrome 79.0 however functional testing will only run on this new version since chrome 79.0 is dropped for functional testing in https://github.com/prebid/Prebid.js/blob/master/wdio.conf.js

### CODE REVIEW

**Comment by ChrisHuie** [comment]
reference issue #6301

**Comment by patmmccann** [comment]
Since this passed tests,  are you planning to add other changes here or do them each in a new pr?

**Comment by ChrisHuie** [comment]
@patmmccann I was planning to do a separate pr for at least the browsers that are running through the functional tests

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
@ChrisHuie I think this requires a second review, added @robertrmartinez to take a look if he's available

**Review by robertrmartinez — APPROVED** [review]

**Pull request merged**
